monailabel/tasks/infer/prompt_utils.py

```diff
@@ -0,0 +1,136 @@
+from typing import Any, Dict
```

Comment:
when vista3d bundle is used.. why we need to have this related code in monailabel?